tests/Unit/Db/AuthorizationCodeTest.php

```diff
@@ -36,7 +36,7 @@ public function setUp(): void {
 	public function testResetExpires() {
 		$expected = \time() + AuthorizationCode::EXPIRATION_TIME;
 		$this->authorizationCode->resetExpires();
-		$this->assertEquals($expected, $this->authorizationCode->getExpires(), '', 1);
+		$this->assertEqualsWithDelta($expected, $this->authorizationCode->getExpires(), '', 1);
```

Comment:
```suggestion
		$this->assertEqualsWithDelta($expected, $this->authorizationCode->getExpires(), 1);
```
The delta is the 3rd parameter. Maybe code should be like this?